simphony_metadata.yml

```diff
@@ -0,0 +1,398 @@
+# indicates a directional first order relation.
+
+# CUBA.KEY (KEY can be any data key) are attributes intrinsic to the
+# data entity defined.
+
+# Each CUBA.KEY has its own definition as an entity, which defines its
+# type (parent) in a unique universal manner.
+
+# Each CUBA.KEY can have additional implementation-related metadata,
+# these metadata are the following:
+
+# default: a default value, when absent, no default is implied and the
+# user has to eventually supply a value (not necessarily upon
+# instantiation, implementation dependent).
+ 
+# If omitted, no default is assumed, and the user has to supply a
+# value.
+
+# scope: CUBA.USER or CUBA.SYSTEM.  if omitted, the CUBA.USER is
+# assumed by default
+
+# shape: as [i,j,] specifying that it is a sequence of entities of its
+# own type.  [i, j] means an array of i*j types.  [i, ] means the
+# sequence or array can have minimum of i elements. [, i] the array can
+# have any, maximum j elements. If omitted, a shape of [1] is assumed.
+
+# The YAML is a hierarchical, without lists, to be easily transformed
+# into Python dictionary structure without lists.
+
+CUDS_KEYS:
+
+# basic metadata item, has a UUID and a data container. a data
+# container is a lower level syntactic and semantic metadata schema
+# that defines cross domain entities that are common basic universal
+# attributes CUBA.
+
+  CUDS_ITEM:
+    parent: ""
+    CUBA.UUID:
+      scope: CUBA.SYSTEM
+    CUBA.DATA:  
+      
+# basic cuds component, basis for all CUDS components as prescribed in
+# the SimPhoNy D1.6 specifications.
+
+  CUDS_COMPONENT:
+    definition: Base data type for the CUDS components
+    parent: CUBA.CUDS_ITEM
+    CUBA.DESCRIPTION:
+      default: "" 
+      scope: CUBA.USER
+    CUBA.NAME:
+      default: ""
+      scope: CUBA.USER
+
+# define a top level computational model of SimPhoNy
+  COMPUTATIONAL_MODEL: 
+    definition: SimPhoNy model category according to the RoMM
+    parent: CUBA.CUDS_COMPONENT
+
+  ELECTRONIC:
+    definition: SimPhoNy electronic model category according to the RoMM
+    parent: CUBA.COMPUTATIONAL_MODEL
+
+  ATOMISTIC:
+    definition: SimPhoNy atomistic model category according to the RoMM
+    parent: CUBA.COMPUTATIONAL_MODEL
+
+  MESOSCOPIC:
+    definition: SimPhoNy mesoscopic model category according to the RoMM
+    parent: CUBA.COMPUTATIONAL_MODEL
+
+  CONTINUUM:
+    definition: SimPhoNy continuum model category according to the RoMM
+    parent: CUBA.COMPUTATIONAL_MODEL
+ 
+  MODEL_EQUATION:
+    definition: SimPhoNy Model Equation = PE + MRs according to the RoMM
+    models: []
+    parent: CUBA.CUDS_COMPONENT
+    physics_variables: []
+
+  PHYSICS_EQUATION:
+   definition: SimPhoNy Physics Equation
+   parent: CUBA.MODEL_EQUATION
+
+  MATERIAL_RELATION_EQUATION:
+    definition: SimPhoNy Material Relation
+    parent: CUBA.MODEL_EQUATION
+    CUBA.MATERIAL:
+      scope: CUBA.USER
+      shape: [1,]
```

Comment:
Why is it necessary to have `shape: [1,]` here while other places have simply `[]`?